engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql

```diff
@@ -307,204 +307,6 @@ CREATE TABLE `cloud`.`shared_filesystem`(
     INDEX `i_shared_filesystem__domain_id`(`domain_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
-DROP VIEW IF EXISTS `cloud`.`user_vm_view`;
```

Comment:
@abh1sar this view updated for shared filesystem? not needed in upgrade path?

Reply:
@sureshanaparti yes, the view is created in shared_filesystem_view.sql
We don't put any views in the upgrade path anymore.

View for user_vm_view was also udpdated in user_vm_view.sql. It was not needed here.